### ISSUE TITLE
request for update template GoogleSitemapController_sitemap.ss

### ISSUE THREAD

**Issue by guyvanbael** [tractorcow/silverstripe-fluent#376]
yesterday, Andrew and I fixed the sitemapmodule issues with SS 4.1
Could you pls update the fluent template to this:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type='text/xsl' href='{$BaseHref}sitemap.xml/styleSheet'?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1">
	<% loop $Items %>
        <url>
            <loc>$AbsoluteLink</loc>
            <% if $LastEdited %><lastmod>$LastEdited.Rfc3339()</lastmod><% end_if %>
            <% if $ChangeFrequency %><changefreq>$ChangeFrequency</changefreq><% end_if %>
            <% if $GooglePriority %><priority>$GooglePriority</priority><% end_if %>
            <% if $Locales %><% loop $Locales %><% if $LinkingMode != 'invalid' %>
            <xhtml:link
                rel="alternate"
                hreflang="$LocaleRFC1766"
                href="$AbsoluteLink"
                />
            <% end_if %><% end_loop %><% end_if %>
            <% if $ImagesForSitemap %><% loop $ImagesForSitemap %>
                <image:image>
                    <image:loc>{$AbsoluteLink}</image:loc>
                </image:image>
            <% end_loop %><% end_if %>
        </url>
	<% end_loop %>
</urlset>
```

**Comment by robbieaverill** [comment]
Done in #375?

**Comment by guyvanbael** [comment]
In version 2.1.2 the images are added

**Comment by robbieaverill** [comment]
Ah right! Would you like to make a PR to update it here?

**Comment by guyvanbael** [comment]
never done this.... gonna give it a try right away...

**Comment by guyvanbael** [comment]
@robbieaverill Pull Request is done.

**Comment by dhensby** [comment]
fixed by #377

**Comment by robbieaverill** [comment]
Thanks @guyvanbael, nice job :)
